libsoftwarecontainer/lxc-softwarecontainer.in

```diff
@@ -132,6 +132,7 @@ while true
 do
     case "$1" in
         -h|--help)      usage $0 && exit 0;;
+        -b|--buffer)    buffer="true";;
```

Comment:
It looks like the buffer variable is not used anywhere inside the template, so what is it doing there?